executor/runtime/docker/docker.go

```diff
@@ -2218,6 +2218,29 @@ func getExtraContainerEntrypoint(c *runtimeTypes.ExtraContainer) []string {
 	return c.ImageInspect.ContainerConfig.Entrypoint
 }
 
+func computeExtraContainersDockerCommand(c *runtimeTypes.ExtraContainer) []string {
+	if len(c.V1Container.Args) > 0 {
+		// If explicitly set k8s Args (docker CMD) on the pod, we need to return that, no question.
+		// This will be used as the CMD for the ENTRYPOINT already configured (which come from either the pod or the image)
+		return c.V1Container.Args
+	}
+
+	if len(c.V1Container.Command) > 0 {
+		// If there was a k8s Command (docker ENTRYPOINT) specified on the pod, then it would be inappropriate
```

Comment:
It's so confusing that `Command` refers to entrypoint rather than the args (`CMD`) here :(.